lib/screens/wods/add_wod.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+//my imports
+import 'package:kabod_app/core/presentation/constants.dart';
+import 'package:kabod_app/core/presentation/routes.dart';
+import 'package:kabod_app/screens/wods/components/add_wod_form.dart';
+
+class AddWodScreen extends StatefulWidget {
+  final DateTime selectedDay;
+  AddWodScreen({this.selectedDay});
+  @override
+  _AddWodScreenState createState() => _AddWodScreenState();
+}
+
+class _AddWodScreenState extends State<AddWodScreen> {
+  final _formKey = GlobalKey<FormBuilderState>();
+  var wodTypeOptions = [
+    'For Time',
+    'For Reps and Time',
+    'For Weight',
+    'AMRAP',
+    'Custom'
+  ];
```

Comment:
`wodTypeOptions` should be used as an enumerator here and this is what only enums should be saved as wod_type on the database to make sure that no plain strings are inserted.

Take a look at the enum documentation to understand it a little better:
https://dart.dev/guides/language/language-tour#enumerated-types